docs/admin/authorization.md

```diff
@@ -115,6 +115,20 @@ To permit an action Policy with an unset namespace applies regardless of namespa
 
 [Complete file example](http://releases.k8s.io/HEAD/pkg/auth/authorizer/abac/example_policy_file.jsonl)
 
+### A quick note on service accounts
+
+A service account automatically generates a user. The user's name is generated according to the naming convention:
+
+```
+system:serviceaccount:<namespace>:<serviceaccountname>
+```
```

Comment:
Add:
*Creating a new namespace also causes a new service account to be created, of this form:
```
system:serviceaccount:<namespace>:default
```